magento/magento2
# Remove extraneous whitespace - #26275

### Description (*)

This PR removes the extraneous spaces from the labels.

### Fixed Issues (if relevant)

1. magento/magento2#26275

### Manual testing scenarios (*)

1. Go to checkout
2. There should not be any extra whitespace before the asterisk / required icon.

Without this PR:
![grafik](https://user-images.githubusercontent.com/827205/71877331-20af4900-3129-11ea-940b-ce4ad2024a97.png)

With the fixes in the PR:
![grafik](https://user-images.githubusercontent.com/827205/71877560-bb0f8c80-3129-11ea-8b2d-5d9e894c4f99.png)


### Questions or comments


### Contribution checklist (*)
